src/scopes/activityReport/index.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/prefer-default-export */
+import { map, pickBy } from 'lodash';
+
+import { withGranteeName, withoutGranteeName } from './grantee';
+import { withoutReportIds, withReportIds } from './reportId';
+import { beforeStartDate, afterStartDate, withinStartDates } from './startDate';
+import { withoutTopics, withTopics } from './topic';
+import { beforeLastSaveDate, afterLastSaveDate, withinLastSaveDates } from './updatedAt';
+import { withAuthor, withoutAuthor } from './author';
+import { withCollaborators, withoutCollaborators } from './collaborators';
+
```

Comment:
Did we determine that there is no need to filter by status?

Reply:
The only place filtering by status would be helpful is in the "My Alerts" table. Along with your comment

> the my alerts table can be filtered by columns that are not part of the table. Assuming that's what we want. Asking since users my be a bit confused. On the other hand though, it is nice they can do that.

Seems like each table should have it's own filtering criteria